roadmap.md

```diff
@@ -95,5 +95,5 @@ This scenario loads a web page and prints it as a PDF. In spec terms, this invol
 
 This scenario allows to emulate a device, e.g. a mobile phone, and its capabilities, e.g. screen size and pixel density. In spec terms, this involves:
 - [x] Some items from the previous milestones
-- [ ] [Emulating width, height, deviceScaleFactor of the view port](https://github.com/w3c/webdriver-bidi/issues/415)
+- [ ] [Changing the viewport width and height](https://github.com/w3c/webdriver-bidi/issues/415)
```

Comment:
I think that we should still keep the deviceScaleFactor in this list but just as another item. Maybe you can file an issue for that so it can directly be linked?